kinto2xml/exporter.py

```diff
@@ -7,38 +7,59 @@
 logger = logging.getLogger("kinto2xml")
 
 
-def build_version_range(root, item):
+def build_version_range(root, item, app_id, app_ver=None):
     for version in item['versionRange']:
-        versionRange = etree.SubElement(
-            root, 'versionRange')
-        minVersion = version.get('minVersion')
-        if minVersion:
-            versionRange.set('minVersion', minVersion)
+        if (not version.get('targetApplication') or
+                any(tA for tA in version.get('targetApplication', [])
+                    if not tA.get('guid') or tA.get('guid') == app_id)):
```

Comment:
you know my appetite for intermediary variables :) this one is though to digest !